content/getting-started/electricalEngeneering.md

```diff
@@ -68,6 +68,18 @@ Do not use the USB port of the computer to power the electronics. Errors in the
 In case of an additional power supply, the ground of the Raspberry Pi and the ground of the power supply must be connected to each other.
 {{% /notice %}}
 
+## Breadboard
+[Here](https://www.youtube.com/watch?v=6WReFkfrUIk) is a short video that explains the basics of breadboards. 
+{{< gallery >}}
+{{< figure link="/assets/getting-started/electricalEngineering/breadboard.jpeg" caption="Breadboard" caption-position="center" caption-effect="fade" >}}
+{{< /gallery >}}
+{{< load-photoswipe >}}
+
+{{% notice note %}}
+The red and blue lines on the side show which pins are connected. On some boards these lines are interrupted in the middle (see picture breadboard). 
```

Comment:
Maybe extra note: red and blue can also be swapped, not all suppliers use the same position...

Reply:
done